tests/openvino/test_modeling.py

```diff
@@ -611,6 +613,59 @@ def test_default_filling_attention_mask(self):
         del model_with_cache
         gc.collect()
 
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    @unittest.skipIf(not IS_SUPPORT_STATEFUL, "Stateful models supported only in 2023.3 and above")
+    def test_stateful(self, model_arch):
```

Comment:
it would be helpful to have a quick unit test for weight compression applied to the stateful model as well.

Reply:
added